### PR TITLE
Update Keybase completions

### PR DESCRIPTION
**Pull request by 239**
## Description

Added global options for Keybase and updated some commands.

Fixes issue #

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [x] User-visible changes noted in CHANGELOG.md

### CODE REVIEW

File: share/completions/keybase.fish [hunk 1]
```diff
@@ -1,70 +1,156 @@
-#Keybase 5.0.0
+#Keybase 5.1.1
 
-function __fish_keybase_commandline_ends_with
-    set -l line (commandline -poc)
+function __fish_keybase_line_ends_with
+    set -l line
+    for s in (commandline -poc)
+        set -l c (string sub -l 1 -- $s)
+        if test "$c" != "-"; set -a line $s; end
```
Comment (krobelus) [inline share/completions/keybase.fish]: You can do this in one line:
`set -l line (commandline -poc | string match -v -r '^-')`

Comment (239) [reply share/completions/keybase.fish]: I avoided regex for no reason, after a quick test it seems there is almost no difference.
That one line is much more elegant, thanks!

File: fish.desktop [hunk 2]
```diff
@@ -1,12 +0,0 @@
-[Desktop Entry]
```
Comment (faho) [inline fish.desktop]: You merged/rebased this wrong. That file was added between you opening the PR and adding the second commit.

File: sphinx_doc_src/faq.rst [hunk 3]
```diff
@@ -14,7 +14,7 @@ Since fish 3.1 you can set an environment variable for just one command using th
     begin; set -lx key value; echo $key; end
 
 How do I run a command every login? What's fish's equivalent to .bashrc or .profile?
-------------------------------------------------------------------------------------
+------------------------------------------------------------------------
```
Comment (faho) [inline sphinx_doc_src/faq.rst]: Similarly, this is undoing changes from between the two commits. I'd suggest something like `git reset --soft HEAD^`, then adding *just your changes*, redoing the commit and `git push --force` (yes, this needs force).

Comment (239) [reply sphinx_doc_src/faq.rst]: Sorry, I wasn't paying attention while pulling from the wrong remote.
This is what happens when one is too lazy to make a separate branch.

**Comment by faho** [comment]
Merged, thanks!

**Pull request merged**